DockerSettings.php

```diff
@@ -409,7 +409,11 @@
 //}
 
 ######################### Custom Settings ##########################
-@include( 'CustomSettings.php' );
+if ( file_exists( "$IP/_settings/LocalSettings.php" ) ) {
+	require_once "$IP/_settings/LocalSettings.php";
+} elseif ( file_exists( "$IP/CustomSettings.php" ) ) {
+	require_once "$IP/CustomSettings.php";
+}
```

Comment:
Same as for https://github.com/WikiTeq/docker-wikizilla.org/pull/12/files I assume this is leaked from another PR, not treating as a blocker though